yt_dlp/extractor/facebook.py

```diff
@@ -420,6 +421,26 @@ def extract_metadata(webpage):
                 r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
             post = traverse_obj(post_data, (
                 ..., 'require', ..., ..., ..., '__bbox', 'require', ..., ..., ..., '__bbox', 'result', 'data'), expected_type=dict) or []
+            snippet = traverse_obj(post, (..., 'video', ..., 'attachments', ..., lambda k, v: (
+                k == 'media' and str(v['id']) == video_id and v['__typename'] == 'Video')))
+            locale = self._html_search_meta(['og:locale', 'twitter:locale'], webpage, 'locale', default='en_US')
+            captions = get_first(snippet, 'video_available_captions_locales', 'captions_url')
+            is_video_broadcast = get_first(snippet, 'is_video_broadcast', expected_type=bool)
+            automatic_captions = {}
+            subtitles = {}
+            if url_or_none(captions):  # snippet only had 'captions_url'
+                subtitles[locale] = [{'url': captions}]
+            else:
+                captions = sorted(captions, key=lambda c: c['locale'])
+            for caption in traverse_obj(captions, lambda _, v: v['captions_url'] and v['locale']):
```

Comment:
gotta make this non-fatal though
```suggestion
            for caption in traverse_obj(captions, (
                {lambda x: sorted(x, key=lambda c: c['locale'])}, lambda _, v: v['captions_url'])
            ):
```